server/test/expand.go

```diff
@@ -86,22 +71,78 @@ func TestExpandQuery(t *testing.T, datastore storage.OpenFGADatastore) {
 			},
 		},
 		{
-			name: "computed userset",
-			typeDefinitions: []*openfgapb.TypeDefinition{
+			name: "1.1_simple_direct",
+			model: &openfgapb.AuthorizationModel{
+				Id:            ulid.Make().String(),
+				SchemaVersion: typesystem.SchemaVersion1_1,
+				TypeDefinitions: []*openfgapb.TypeDefinition{
+					{
+						Type: "user",
+					},
+					{
+						Type: "repo",
+						Relations: map[string]*openfgapb.Userset{
+							"admin": typesystem.This(),
+						},
+						Metadata: &openfgapb.Metadata{
+							Relations: map[string]*openfgapb.RelationMetadata{
+								"admin": {
+									DirectlyRelatedUserTypes: []*openfgapb.RelationReference{
+										{
+											Type: "user",
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			tuples: []*openfgapb.TupleKey{
 				{
-					Type: "repo",
-					Relations: map[string]*openfgapb.Userset{
-						"admin": {},
-						"writer": {
-							Userset: &openfgapb.Userset_ComputedUserset{
-								ComputedUserset: &openfgapb.ObjectRelation{
-									Relation: "admin",
+					Object:   "repo:openfga/foo",
+					Relation: "admin",
+					User:     "user:jon",
+				},
+			},
+			request: &openfgapb.ExpandRequest{
+				TupleKey: &openfgapb.TupleKey{
```

Comment:
Here and elsewhere would you please change these to `tuple.NewTupleKey("object", "relation", "user")` (for example).

Reply:
Normally i'd agree but Expand in particular does not need the `User` parameter 🤷‍♀️

Reply:
So use `""`